CITATION.cff

```diff
@@ -0,0 +1,23 @@
+# Parser settings.
+cff-version: 1.2.0
+message: Please cite this crate using these information.
+
+# Version information.
+date-released: 2022-11-12
+version: 0.4.23
```

Comment:
Release date and version are optional.  GitHub will add these information to the rendered citation metadata, if specified.  For the maintenance of the version, I prefer [bump2version](https://github.com/c4urself/bump2version).  For the automatic release date update on release, I created [this GitHub Action](https://github.com/kevinmatthes/cff-release-today).

Reply:
It's not super-clear to me how the Action works (when does it update the date?).

Reply:
It's all manual. I think I'd be okay to stick with manual updates for this file too, for now.

Reply:
Okay, no problem.